sympy/core/relational.py

```diff
@@ -109,13 +119,16 @@ def canonical(self):
         """
         r = self
         if r.func in (Ge, Gt):
-            r = r.reversed
+            if _coeff_isneg(r.rhs):
```

Comment:
I realize that it is not easy to devise a good plan for canonicalization of inequalities. Right now I can only propose simple (and obviously inadequate) rules such as not changing the direction if the LHS is a single symbol. (It would probably be better if one could tell variable and constant symbols apart.)